library/classes/thumbnail/Thumbnail.class.php

```diff
@@ -32,6 +32,12 @@ class Thumbnail
      */
     public function __construct($max_size = null)
     {
+        //validate that GD extension was enabled
+        if (!extension_loaded('gd'))
+        {
+            die('Abort '.basename(__FILE__).' : Missing DG extensions');
```

Comment:
Will the user see this error?
(Also recommend sending an error message to the php error log)